source/Core/Threads/GUIThread.cpp

```diff
@@ -713,7 +713,7 @@ void showDebugMenu(void) {
       break;
     case 10:
       // Print PCB ID number
```

Comment:
This should read:
// Print ACC type
Am I right? 😁

Reply:
Eugh yeah
missed that 🙃

Reply:
Sorry was a bit too late. 😅